INFO:root:predictions: [7, 2, 1, 0, 4, 1, 4, 9, 5, 9, 0, 6, 9, 0, 1, 5, 9, 7, 8, 4, 9, 6, 6, 5, 4, 0, 7, 4, 0, 1, 3, 1]
INFO:root:labels: [7 2 1 0 4 1 4 9 5 9 0 6 9 0 1 5 9 7 3 4 9 6 6 5 4 0 7 4 0 1 3 1]
INFO:root:test loss: 0.048442352563142776, test accuracy: 0.96875
INFO:root:predictions: [7, 2, 1, 0, 4, 1, 4, 9, 5, 9, 0, 6, 9, 0, 1, 5, 9, 7, 8, 4, 9, 6, 6, 5, 4, 0, 7, 4, 0, 1, 3, 1]
INFO:root:labels: [7 2 1 0 4 1 4 9 5 9 0 6 9 0 1 5 9 7 3 4 9 6 6 5 4 0 7 4 0 1 3 1]
INFO:root:predictions: [7, 2, 1, 0, 4, 1, 4, 9, 5, 9, 0, 6, 9, 0, 1, 5, 9, 7, 8, 4, 9, 6, 6, 5, 4, 0, 7, 4, 0, 1, 3, 1]
INFO:root:labels: [3 4 7 2 7 1 2 1 1 7 4 2 3 5 1 2 4 4 6 3 5 5 6 0 4 1 9 5 7 8 9 3]
INFO:root:test loss: 5.005849361419678, test accuracy: 0.546875
INFO:root:predictions: [7, 2, 1, 0, 4, 1, 4, 9, 5, 9, 0, 6, 9, 0, 1, 5, 9, 7, 8, 4, 9, 6, 6, 5, 4, 0, 7, 4, 0, 1, 3, 1]
INFO:root:labels: [7 2 1 0 4 1 4 9 5 9 0 6 9 0 1 5 9 7 3 4 9 6 6 5 4 0 7 4 0 1 3 1]
INFO:root:predictions: [7, 2, 1, 0, 4, 1, 4, 9, 5, 9, 0, 6, 9, 0, 1, 5, 9, 7, 8, 4, 9, 6, 6, 5, 4, 0, 7, 4, 0, 1, 3, 1]
INFO:root:labels: [3 4 7 2 7 1 2 1 1 7 4 2 3 5 1 2 4 4 6 3 5 5 6 0 4 1 9 5 7 8 9 3]
INFO:root:predictions: [7, 2, 1, 0, 4, 1, 4, 9, 5, 9, 0, 6, 9, 0, 1, 5, 9, 7, 8, 4, 9, 6, 6, 5, 4, 0, 7, 4, 0, 1, 3, 1]
INFO:root:labels: [7 4 6 4 3 0 7 0 2 9 1 7 3 2 9 7 7 6 2 7 8 4 7 3 6 1 3 6 9 3 1 4]
INFO:root:test loss: 7.184318542480469, test accuracy: 0.3854166567325592
INFO:root:predictions: [7, 2, 1, 0, 4, 1, 4, 9, 5, 9, 0, 6, 9, 0, 1, 5, 9, 7, 8, 4, 9, 6, 6, 5, 4, 0, 7, 4, 0, 1, 3, 1]
INFO:root:labels: [7 2 1 0 4 1 4 9 5 9 0 6 9 0 1 5 9 7 3 4 9 6 6 5 4 0 7 4 0 1 3 1]
INFO:root:predictions: [7, 2, 1, 0, 4, 1, 4, 9, 5, 9, 0, 6, 9, 0, 1, 5, 9, 7, 8, 4, 9, 6, 6, 5, 4, 0, 7, 4, 0, 1, 3, 1]
INFO:root:labels: [3 4 7 2 7 1 2 1 1 7 4 2 3 5 1 2 4 4 6 3 5 5 6 0 4 1 9 5 7 8 9 3]
INFO:root:predictions: [7, 2, 1, 0, 4, 1, 4, 9, 5, 9, 0, 6, 9, 0, 1, 5, 9, 7, 8, 4, 9, 6, 6, 5, 4, 0, 7, 4, 0, 1, 3, 1]
INFO:root:labels: [7 4 6 4 3 0 7 0 2 9 1 7 3 2 9 7 7 6 2 7 8 4 7 3 6 1 3 6 9 3 1 4]
INFO:root:predictions: [7, 2, 1, 0, 4, 1, 4, 9, 5, 9, 0, 6, 9, 0, 1, 5, 9, 7, 8, 4, 9, 6, 6, 5, 4, 0, 7, 4, 0, 1, 3, 1]
INFO:root:labels: [1 7 6 9 6 0 5 4 9 9 2 1 9 4 8 7 3 9 7 4 4 4 9 2 5 4 7 6 7 9 0 5]
INFO:root:test loss: 7.938594818115234, test accuracy: 0.3203125
INFO:root:predictions: [7, 2, 1, 0, 4, 1, 4, 9, 5, 9, 0, 6, 9, 0, 1, 5, 9, 7, 8, 4, 9, 6, 6, 5, 4, 0, 7, 4, 0, 1, 3, 1]
INFO:root:labels: [7 2 1 0 4 1 4 9 5 9 0 6 9 0 1 5 9 7 3 4 9 6 6 5 4 0 7 4 0 1 3 1]
INFO:root:predictions: [7, 2, 1, 0, 4, 1, 4, 9, 5, 9, 0, 6, 9, 0, 1, 5, 9, 7, 8, 4, 9, 6, 6, 5, 4, 0, 7, 4, 0, 1, 3, 1]
INFO:root:labels: [3 4 7 2 7 1 2 1 1 7 4 2 3 5 1 2 4 4 6 3 5 5 6 0 4 1 9 5 7 8 9 3]
INFO:root:predictions: [7, 2, 1, 0, 4, 1, 4, 9, 5, 9, 0, 6, 9, 0, 1, 5, 9, 7, 8, 4, 9, 6, 6, 5, 4, 0, 7, 4, 0, 1, 3, 1]
INFO:root:labels: [7 4 6 4 3 0 7 0 2 9 1 7 3 2 9 7 7 6 2 7 8 4 7 3 6 1 3 6 9 3 1 4]
INFO:root:predictions: [7, 2, 1, 0, 4, 1, 4, 9, 5, 9, 0, 6, 9, 0, 1, 5, 9, 7, 8, 4, 9, 6, 6, 5, 4, 0, 7, 4, 0, 1, 3, 1]
INFO:root:labels: [1 7 6 9 6 0 5 4 9 9 2 1 9 4 8 7 3 9 7 4 4 4 9 2 5 4 7 6 7 9 0 5]
INFO:root:predictions: [7, 2, 1, 0, 4, 1, 4, 9, 5, 9, 0, 6, 9, 0, 1, 5, 9, 7, 8, 4, 9, 6, 6, 5, 4, 0, 7, 4, 0, 1, 3, 1]
INFO:root:labels: [8 5 6 6 5 7 8 1 0 1 6 4 6 7 3 1 7 1 8 2 0 2 9 9 5 5 1 5 6 0 3 4]
INFO:root:test loss: 8.723630905151367, test accuracy: 0.26875001192092896
INFO:root:predictions: [7, 2, 1, 0, 4, 1, 4, 9, 5, 9, 0, 6, 9, 0, 1, 5, 9, 7, 8, 4, 9, 6, 6, 5, 4, 0, 7, 4, 0, 1, 3, 1]
INFO:root:labels: [7 2 1 0 4 1 4 9 5 9 0 6 9 0 1 5 9 7 3 4 9 6 6 5 4 0 7 4 0 1 3 1]
INFO:root:predictions: [7, 2, 1, 0, 4, 1, 4, 9, 5, 9, 0, 6, 9, 0, 1, 5, 9, 7, 8, 4, 9, 6, 6, 5, 4, 0, 7, 4, 0, 1, 3, 1]
INFO:root:labels: [3 4 7 2 7 1 2 1 1 7 4 2 3 5 1 2 4 4 6 3 5 5 6 0 4 1 9 5 7 8 9 3]
INFO:root:predictions: [7, 2, 1, 0, 4, 1, 4, 9, 5, 9, 0, 6, 9, 0, 1, 5, 9, 7, 8, 4, 9, 6, 6, 5, 4, 0, 7, 4, 0, 1, 3, 1]
INFO:root:labels: [7 4 6 4 3 0 7 0 2 9 1 7 3 2 9 7 7 6 2 7 8 4 7 3 6 1 3 6 9 3 1 4]
INFO:root:predictions: [7, 2, 1, 0, 4, 1, 4, 9, 5, 9, 0, 6, 9, 0, 1, 5, 9, 7, 8, 4, 9, 6, 6, 5, 4, 0, 7, 4, 0, 1, 3, 1]
INFO:root:labels: [1 7 6 9 6 0 5 4 9 9 2 1 9 4 8 7 3 9 7 4 4 4 9 2 5 4 7 6 7 9 0 5]
INFO:root:predictions: [7, 2, 1, 0, 4, 1, 4, 9, 5, 9, 0, 6, 9, 0, 1, 5, 9, 7, 8, 4, 9, 6, 6, 5, 4, 0, 7, 4, 0, 1, 3, 1]
INFO:root:labels: [8 5 6 6 5 7 8 1 0 1 6 4 6 7 3 1 7 1 8 2 0 2 9 9 5 5 1 5 6 0 3 4]
INFO:root:predictions: [7, 2, 1, 0, 4, 1, 4, 9, 5, 9, 0, 6, 9, 0, 1, 5, 9, 7, 8, 4, 9, 6, 6, 5, 4, 0, 7, 4, 0, 1, 3, 1]
INFO:root:labels: [4 6 5 4 6 5 4 5 1 4 4 7 2 3 2 7 1 8 1 8 1 8 5 0 8 9 2 5 0 1 1 1]
INFO:root:test loss: 9.145442962646484, test accuracy: 0.2447916716337204
WARNING:tensorflow:Model was constructed with shape (None, 28, 28, 1) for input Tensor("conv2d_input:0", shape=(None, 28, 28, 1), dtype=float32), but it was called on an input with incompatible shape (32, 10).
ERROR:grpc._server:Exception calling application: input tensor must have rank 4
Traceback (most recent call last):
  File "/home/greenmon/.pyenv/versions/tensorflow-env/lib/python3.7/site-packages/tensorflow/python/framework/tensor_shape.py", line 930, in merge_with
    self.assert_same_rank(other)
  File "/home/greenmon/.pyenv/versions/tensorflow-env/lib/python3.7/site-packages/tensorflow/python/framework/tensor_shape.py", line 985, in assert_same_rank
    (self, other))
ValueError: Shapes (32, 10) and (None, None, None, None) must have the same rank

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/greenmon/.pyenv/versions/tensorflow-env/lib/python3.7/site-packages/tensorflow/python/framework/tensor_shape.py", line 1015, in with_rank
    return self.merge_with(unknown_shape(rank=rank))
  File "/home/greenmon/.pyenv/versions/tensorflow-env/lib/python3.7/site-packages/tensorflow/python/framework/tensor_shape.py", line 936, in merge_with
    raise ValueError("Shapes %s and %s are not compatible" % (self, other))
ValueError: Shapes (32, 10) and (None, None, None, None) are not compatible

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/greenmon/.pyenv/versions/tensorflow-env/lib/python3.7/site-packages/tensorflow/python/ops/nn_ops.py", line 1039, in __init__
    input_shape.with_rank(num_spatial_dims + 2)
  File "/home/greenmon/.pyenv/versions/tensorflow-env/lib/python3.7/site-packages/tensorflow/python/framework/tensor_shape.py", line 1017, in with_rank
    raise ValueError("Shape %s must have rank %d" % (self, rank))
ValueError: Shape (32, 10) must have rank 4

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/greenmon/.pyenv/versions/tensorflow-env/lib/python3.7/site-packages/grpc/_server.py", line 435, in _call_behavior
    response_or_iterator = behavior(argument, context)
  File "inference.py", line 54, in process_tensor
    result = self.model.process_data(parsed)
  File "inference.py", line 106, in process_data
    result = self.model(input_data)
  File "/home/greenmon/.pyenv/versions/tensorflow-env/lib/python3.7/site-packages/tensorflow/python/keras/engine/base_layer.py", line 968, in __call__
    outputs = self.call(cast_inputs, *args, **kwargs)
  File "/home/greenmon/.pyenv/versions/tensorflow-env/lib/python3.7/site-packages/tensorflow/python/keras/engine/sequential.py", line 277, in call
    return super(Sequential, self).call(inputs, training=training, mask=mask)
  File "/home/greenmon/.pyenv/versions/tensorflow-env/lib/python3.7/site-packages/tensorflow/python/keras/engine/network.py", line 719, in call
    convert_kwargs_to_constants=base_layer_utils.call_context().saving)
  File "/home/greenmon/.pyenv/versions/tensorflow-env/lib/python3.7/site-packages/tensorflow/python/keras/engine/network.py", line 888, in _run_internal_graph
    output_tensors = layer(computed_tensors, **kwargs)
  File "/home/greenmon/.pyenv/versions/tensorflow-env/lib/python3.7/site-packages/tensorflow/python/keras/engine/base_layer.py", line 968, in __call__
    outputs = self.call(cast_inputs, *args, **kwargs)
  File "/home/greenmon/.pyenv/versions/tensorflow-env/lib/python3.7/site-packages/tensorflow/python/keras/layers/convolutional.py", line 201, in call
    data_format=self._conv_op_data_format)
  File "/home/greenmon/.pyenv/versions/tensorflow-env/lib/python3.7/site-packages/tensorflow/python/ops/nn_ops.py", line 1042, in __init__
    "input tensor must have rank %d" % (num_spatial_dims + 2))
ValueError: input tensor must have rank 4
ERROR:grpc._server:Exception calling application: <_InactiveRpcError of RPC that terminated with:
	status = StatusCode.UNKNOWN
	details = "Exception calling application: input tensor must have rank 4"
	debug_error_string = "{"created":"@1592890146.791984328","description":"Error received from peer ipv6:[::1]:50052","file":"src/core/lib/surface/call.cc","file_line":1056,"grpc_message":"Exception calling application: input tensor must have rank 4","grpc_status":2}"
>
Traceback (most recent call last):
  File "/home/greenmon/.pyenv/versions/tensorflow-env/lib/python3.7/site-packages/grpc/_server.py", line 435, in _call_behavior
    response_or_iterator = behavior(argument, context)
  File "inference.py", line 62, in process_tensor
    request_next_tensor(serialize(result), next_server_address, next_server_port)
  File "inference.py", line 161, in request_next_tensor
    inference_service_pb2.SerializedTensor(data=data))
  File "/home/greenmon/.pyenv/versions/tensorflow-env/lib/python3.7/site-packages/grpc/_channel.py", line 826, in __call__
    return _end_unary_response_blocking(state, call, False, None)
  File "/home/greenmon/.pyenv/versions/tensorflow-env/lib/python3.7/site-packages/grpc/_channel.py", line 729, in _end_unary_response_blocking
    raise _InactiveRpcError(state)
grpc._channel._InactiveRpcError: <_InactiveRpcError of RPC that terminated with:
	status = StatusCode.UNKNOWN
	details = "Exception calling application: input tensor must have rank 4"
	debug_error_string = "{"created":"@1592890146.791984328","description":"Error received from peer ipv6:[::1]:50052","file":"src/core/lib/surface/call.cc","file_line":1056,"grpc_message":"Exception calling application: input tensor must have rank 4","grpc_status":2}"
>
INFO:root:predictions: [7, 2, 1, 0, 4, 1, 4, 9, 5, 9, 0, 6, 9, 0, 1, 5, 9, 7, 8, 4, 9, 6, 6, 5, 4, 0, 7, 4, 0, 1, 3, 1]
INFO:root:labels: [7 2 1 0 4 1 4 9 5 9 0 6 9 0 1 5 9 7 3 4 9 6 6 5 4 0 7 4 0 1 3 1]
INFO:root:test loss: 0.048442352563142776, test accuracy: 0.96875
